plugins/package-managers/swiftpm/src/main/kotlin/SwiftPm.kt

```diff
@@ -95,13 +100,21 @@ class SwiftPm(
      * Resolves dependencies when only a lockfile aka `Package.Resolved` is available. This commonly applies to e.g.
      * Xcode projects which only have a lockfile, but no `Package.swift` file.
      */
-    private fun resolveLockfileDependencies(packageResolvedFile: File): List<ProjectAnalyzerResult> =
-        listOf(
+    private fun resolveLockfileDependencies(packageResolvedFile: File): List<ProjectAnalyzerResult> {
+        val issues = mutableListOf<Issue>()
+
+        val packages = parseLockfile(packageResolvedFile).onFailure {
+            issues += Issue(source = managerName, message = it.message!!)
```

Comment:
With my proposal below, this might catch more exceptions that just our own, so we cannot guarantee that `message`is always non-`null`, and should probably use `.orEmpty()` instead.